src/Model/File.php

```diff
@@ -124,4 +134,16 @@ public function createFromPath(string $path, string $fileName = null): Model
 
         return $entity;
     }
+
+    public function cleanupDrafts(): void
+    {
+        $this->assertIsModel();
```

Comment:
not needed, as the code below requires model on several places

Reply:
addressed